packages/cozy-client/src/models/file.js

```diff
@@ -16,6 +18,29 @@ export const ALBUMS_DOCTYPE = 'io.cozy.photos.albums'
 
 const FILENAME_WITH_EXTENSION_REGEX = /(.+)(\..*)$/
 
+const DEFAULT_NOTICE_PERIOD_DAYS = 90
+const PERSONAL_SPORTING_LICENCE_NOTICE_PERIOD_DAYS = 15
+
+const papersDefinitions = {
+  notifications: {
+    papersToNotify: [
+      {
+        label: 'national_id_card',
+        country: 'fr',
+        expirationDateAttribute: 'expirationDate'
+      },
+      {
+        label: 'residence_permit',
+        expirationDateAttribute: 'expirationDate'
+      },
+      {
+        label: 'personal_sporting_licence',
+        expirationDateAttribute: 'referencedDate'
+      }
+    ]
+  }
+}
```

Comment:
Can you create a file called `paper.js` or something like that and put inside all the stuff related to `paper`? 

Models are very domain oriented. Since here we want to deal with `paper` even if `paper` is a subset of `file` I don't think it should belong to `file` but have its own "domain"